migrations/v2/V202408201553__v2.4.x__ContentAutotagging.sql

```diff
@@ -142,55 +145,55 @@ INSERT INTO [${flyway:defaultSchema}].EntityField ( EntityID, Sequence, Name, Di
 INSERT INTO [${flyway:defaultSchema}].EntityField ( EntityID, Sequence, Name, DisplayName, Description, Type, Length, Precision, Scale, AllowsNull, DefaultValue, AutoIncrement, AllowUpdateAPI, IsVirtual, RelatedEntityID, RelatedEntityFieldName, IsNameField, IncludeInUserSearchAPI, IncludeRelatedEntityNameFieldInBaseView, DefaultInView, IsPrimaryKey, IsUnique, RelatedEntityDisplayType ) VALUES ( 'B420FF22-0E66-EF11-A752-C0A5E8ACCB22', 6, 'URL', 'URL', NULL, 'nvarchar', 4000, 0, 0, 0, 'null', 0, 1, 0, NULL, NULL, 0, 0, 0, 0, 0, 0, 'Search' ) 
-INSERT INTO [${flyway:defaultSchema}].EntityField ( EntityID, Sequence, Name, DisplayName, Description, Type, Length, Precision, Scale, AllowsNull, DefaultValue, AutoIncrement, AllowUpdateAPI, IsVirtual, RelatedEntityID, RelatedEntityFieldName, IsNameField, IncludeInUserSearchAPI, IncludeRelatedEntityNameFieldInBaseView, DefaultInView, IsPrimaryKey, IsUnique, RelatedEntityDisplayType ) VALUES ( 'B420FF22-0E66-EF11-A752-C0A5E8ACCB22', 7, '[${flyway:defaultSchema}]_CreatedAt', 'Created At', NULL, 'datetimeoffset', 10, 34, 7, 0, 'getutcdate()', 0, 0, 0, NULL, NULL, 0, 0, 0, 0, 0, 0, 'Search' ) 
-INSERT INTO [${flyway:defaultSchema}].EntityField ( EntityID, Sequence, Name, DisplayName, Description, Type, Length, Precision, Scale, AllowsNull, DefaultValue, AutoIncrement, AllowUpdateAPI, IsVirtual, RelatedEntityID, RelatedEntityFieldName, IsNameField, IncludeInUserSearchAPI, IncludeRelatedEntityNameFieldInBaseView, DefaultInView, IsPrimaryKey, IsUnique, RelatedEntityDisplayType ) VALUES ( 'B420FF22-0E66-EF11-A752-C0A5E8ACCB22', 8, '[${flyway:defaultSchema}]_UpdatedAt', 'Updated At', NULL, 'datetimeoffset', 10, 34, 7, 0, 'getutcdate()', 0, 0, 0, NULL, NULL, 0, 0, 0, 0, 0, 0, 'Search' ) 
+INSERT INTO [${flyway:defaultSchema}].EntityField ( EntityID, Sequence, Name, DisplayName, Description, Type, Length, Precision, Scale, AllowsNull, DefaultValue, AutoIncrement, AllowUpdateAPI, IsVirtual, RelatedEntityID, RelatedEntityFieldName, IsNameField, IncludeInUserSearchAPI, IncludeRelatedEntityNameFieldInBaseView, DefaultInView, IsPrimaryKey, IsUnique, RelatedEntityDisplayType ) VALUES ( 'B420FF22-0E66-EF11-A752-C0A5E8ACCB22', 7, '${flyway:defaultSchema}_CreatedAt', 'Created At', NULL, 'datetimeoffset', 10, 34, 7, 0, 'getutcdate()', 0, 0, 0, NULL, NULL, 0, 0, 0, 0, 0, 0, 'Search' ) 
+INSERT INTO [${flyway:defaultSchema}].EntityField ( EntityID, Sequence, Name, DisplayName, Description, Type, Length, Precision, Scale, AllowsNull, DefaultValue, AutoIncrement, AllowUpdateAPI, IsVirtual, RelatedEntityID, RelatedEntityFieldName, IsNameField, IncludeInUserSearchAPI, IncludeRelatedEntityNameFieldInBaseView, DefaultInView, IsPrimaryKey, IsUnique, RelatedEntityDisplayType ) VALUES ( 'B420FF22-0E66-EF11-A752-C0A5E8ACCB22', 8, '${flyway:defaultSchema}_UpdatedAt', 'Updated At', NULL, 'datetimeoffset', 10, 34, 7, 0, 'getutcdate()', 0, 0, 0, NULL, NULL, 0, 0, 0, 0, 0, 0, 'Search' ) 
```

Comment:
Looks like find/replace was a little broad

```suggestion
INSERT INTO [${flyway:defaultSchema}].EntityField ( EntityID, Sequence, Name, DisplayName, Description, Type, Length, Precision, Scale, AllowsNull, DefaultValue, AutoIncrement, AllowUpdateAPI, IsVirtual, RelatedEntityID, RelatedEntityFieldName, IsNameField, IncludeInUserSearchAPI, IncludeRelatedEntityNameFieldInBaseView, DefaultInView, IsPrimaryKey, IsUnique, RelatedEntityDisplayType ) VALUES ( 'B420FF22-0E66-EF11-A752-C0A5E8ACCB22', 7, '__mj_CreatedAt', 'Created At', NULL, 'datetimeoffset', 10, 34, 7, 0, 'getutcdate()', 0, 0, 0, NULL, NULL, 0, 0, 0, 0, 0, 0, 'Search' ) 
INSERT INTO [${flyway:defaultSchema}].EntityField ( EntityID, Sequence, Name, DisplayName, Description, Type, Length, Precision, Scale, AllowsNull, DefaultValue, AutoIncrement, AllowUpdateAPI, IsVirtual, RelatedEntityID, RelatedEntityFieldName, IsNameField, IncludeInUserSearchAPI, IncludeRelatedEntityNameFieldInBaseView, DefaultInView, IsPrimaryKey, IsUnique, RelatedEntityDisplayType ) VALUES ( 'B420FF22-0E66-EF11-A752-C0A5E8ACCB22', 8, '__mj_UpdatedAt', 'Updated At', NULL, 'datetimeoffset', 10, 34, 7, 0, 'getutcdate()', 0, 0, 0, NULL, NULL, 0, 0, 0, 0, 0, 0, 'Search' ) 
```

What's more, we can _generally_ omit those timestamp fields from INSERTs in migrations entirely, since they'll be added by the SQL server at whatever time the migration runs.

When I'm using find/replace here, I usually look for `__mj.` or `[__mj].` to avoid overlapping with these timestamp fields.